src/main/java/io/anserini/search/query/QaTopicReader.java

```diff
@@ -0,0 +1,42 @@
+package io.anserini.search.query;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class QaTopicReader extends TopicReader {
```

Comment:
This is only for the TrecQA collection, right? And it's not "topics" in the sense queries? Shouldn't it just be `TrecQAPairs` or something like that?